### ISSUE TITLE
Push Notification with closed browser / app sound not working

### ISSUE THREAD

**Issue by pwFoo** [Ajaxy/telegram-tt#205]
Version "Z" of web client telegram have no "Notification Sound" setting.

![image](https://user-images.githubusercontent.com/333408/205996367-947c6a24-b960-4959-a791-3b13560d042e.png)

Setting is available with "K" Version

![image](https://user-images.githubusercontent.com/333408/205996628-183cd780-23b4-4319-a271-b976bcd43dea.png)

I like the "Z" version because it works fine with Android OS "Share to App" so would be nice to get (offline / closed app notification) sound working :)

**Comment by KnorpelSenf** [comment]
It's called “Web notifications” in Z and you'll find them on the same settings page at the very top. If you enable them, you'll receive notifications about new messages even if Z is closed or in the background.

**Comment by pwFoo** [comment]
Yes, I receive notifications, but without sound. With display off I don't see it. So I need a notification sound to hear it.

**Comment by KnorpelSenf** [comment]
Can you share a screenshot from the very top of that same settings page?

**Comment by pwFoo** [comment]
![Screenshot_20221207-170119](https://user-images.githubusercontent.com/333408/206229141-f2766feb-292d-4247-b68d-64b530e051a3.jpg)

**Comment by pwFoo** [comment]
Telegram web version "K" sound works fine, but "share to" doesn't work. Other apps sound works too.
Don't know if that could be a app specific configuration 🤷‍♂️
